src/__mocks__/handlers/DataExplorer/handlers.ts

```diff
@@ -0,0 +1,761 @@
+    },
+    {
+      '@type': 'DataDownload',
+      atLocation: {
+        '@type': 'Location',
+        location:
+          'file:///gpfs/bbp.cscs.ch/data/project/proj109/nexus/c7d70522-4305-480a-b190-75d757ed9a49/6/4/3/8/3/d/0/3/tkb060126a2_ch3_bc_n_jh_100x_1.swc',
+        store: {
+          '@id':
+            'https://bbp.epfl.ch/neurosciencegraph/data/4820323e-bee0-48d2-824f-9d9d404dbbee',
+          '@type': 'RemoteDiskStorage',
+          _rev: 1,
+        },
+      },
+      contentSize: {
+        unitCode: 'bytes',
+        value: 891821,
+      },
+      contentUrl:
+        'https://bbp.epfl.ch/nexus/v1/files/public/sscx/https%3A%2F%2Fbbp.epfl.ch%2Fneurosciencegraph%2Fdata%2F60025362-1ca8-425e-908c-a01e4661c3e7',
+      digest: {
+        algorithm: 'SHA-256',
+        value:
+          '22bac983b129fe806c80a9ddb4dcf77b79c1a6a28adffd6674290fb1f014a30e',
+      },
+      encodingFormat: 'application/swc',
+      name: 'tkb060126a2_ch3_bc_n_jh_100x_1.swc',
+    },
+  ],
+  generation: {
+    '@type': 'Generation',
+    activity: {
+      '@id':
+        'https://bbp.epfl.ch/neurosciencegraph/data/9ad281da-e352-4275-b1fa-6a3516a654c9',
+      '@type': ['Activity', 'Reconstruction'],
+    },
+  },
+  isPartOf: {
+    '@id':
+      'https://bbp.epfl.ch/neurosciencegraph/data/neuronmorphologies/23d3d87e-94fe-4639-b5c8-a26a712587e6',
+    '@type': 'Entity',
+  },
+  license: {
+    '@id': 'https://creativecommons.org/licenses/by/4.0/',
+    '@type': 'License',
+  },
+  name: 'tkb060126a2_ch3_bc_n_jh_100x_1',
+  objectOfStudy: {
+    '@id':
+      'http://bbp.epfl.ch/neurosciencegraph/taxonomies/objectsofstudy/singlecells',
+    '@type': 'ObjectOfStudy',
+    label: 'Single Cell',
+  },
+  sameAs:
+    'https://bbp.epfl.ch/neurosciencegraph/data/neuronmorphologies/431a1196-47b5-41a2-931a-3577be9a2dc4',
+  subject: {
+    '@type': 'Subject',
+    species: {
+      '@id': 'NCBITaxon:10116',
+      label: 'Rattus norvegicus',
+    },
+  },
+  _constrainedBy: 'https://neuroshapes.org/dash/neuronmorphology',
+  _createdAt: '2021-11-23T11:34:00.952Z',
+  _createdBy: 'https://bbp.epfl.ch/nexus/v1/realms/bbp/users/akkaufma',
+  _deprecated: false,
+  _incoming:
+    'https://bbp.epfl.ch/nexus/v1/resources/public/sscx/datashapes:neuronmorphology/neuronmorphologies%2Fbfdd4d1a-8b06-46fe-b663-7d9f8020dcaf/incoming',
+  _outgoing:
+    'https://bbp.epfl.ch/nexus/v1/resources/public/sscx/datashapes:neuronmorphology/neuronmorphologies%2Fbfdd4d1a-8b06-46fe-b663-7d9f8020dcaf/outgoing',
+  _project: 'https://bbp.epfl.ch/nexus/v1/projects/public/sscx',
+  _rev: 2,
+  _schemaProject:
+    'https://bbp.epfl.ch/nexus/v1/projects/neurosciencegraph/datamodels',
+  _self:
+    'https://bbp.epfl.ch/nexus/v1/resources/public/sscx/datashapes:neuronmorphology/neuronmorphologies%2Fbfdd4d1a-8b06-46fe-b663-7d9f8020dcaf',
+  _updatedAt: '2023-06-23T07:34:56.011Z',
+  _updatedBy: 'https://bbp.epfl.ch/nexus/v1/realms/bbp/users/cgonzale',
+};
+
+const initialResource = {
```

Comment:
Because mock data can be fairly large, it's often a good idea to create utility functions that return them. This way, repetitions of large mock values are removed (there's only one function that contains the whole data). 

If you want, there's already some utility functions (called `getMockResource()`) that can be resued here maybe.

I haven't yet verified if we exclude handler files from the production build, but if we are not excluding them, then we should be specially careful when adding large mock objects.